daemon/daemon.go

```diff
@@ -131,16 +105,46 @@ func (c *Command) canAccess(r *http.Request, user *auth.UserState) bool {
 		return true
 	}
 
-	if r.Method != "GET" {
+	if r.Method == "GET" {
+		// Guest and user access restricted to GET requests
+		if c.GuestOK {
+			return true
+		}
+
+		if isUser && c.UserOK {
+			return true
+		}
+	}
+
+	// Remaining admin checks rely on identifying peer uid
+	if !isUser {
 		return false
 	}
 
-	if isUser && c.UserOK {
+	if uid == 0 {
+		// Superuser does anything.
 		return true
 	}
 
-	if c.GuestOK {
-		return true
+	if c.PolkitOK != "" {
+		allow, err := strconv.ParseBool(r.Header.Get(client.AllowInteractionHeader))
+		if err != nil {
```

Comment:
Can we please fix the change performed in the prior PR just before merging?  The point was to use the default behavior (false) when the header was _missing entirely,_ but keep the original logging if an error happens due to an invalid header.

/cc @mvo5

Reply:
I've pushed a fix for this to add back the logging on garbage values, while not logging when the header is missing.